### PR TITLE
Interning at Deliveroo

### PR DESCRIPTION
**Pull request by jphastings**
Reemma's article about her time here as an intern.

### CODE REVIEW

File: _posts/2017-12-06-interning-at-deliveroo.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+In my second week, I was introduced to my mentor who was currently helping launch an important new feature for Riders in Paris. This meant the opportunity to go to Paris as well, and meet the rest of the team working there! My mentor played a huge role in getting me to start pushing code for Deliveroo. I started off attending retros and working on tickets and I was offered the chance to pair program with my mentor and other members of the team which was an awesome way of getting to know everyone’s roles in the team, and for me, it was a more natural way to getting familiar with the codebase.
```
Comment (jennysivapalan) [inline _posts/2017-12-06-interning-at-deliveroo.md]: Could we add a name to Reemma's mentor?

Comment (jphastings) [reply _posts/2017-12-06-interning-at-deliveroo.md]: Certainly! Who was it?! 😅  @nas?

Comment (nas) [reply _posts/2017-12-06-interning-at-deliveroo.md]: > Certainly! Who was it?! 

@mvalipour

Comment (jphastings) [reply _posts/2017-12-06-interning-at-deliveroo.md]: 👍Will add that here tomorrow.

**Pull request merged**